### DISCUSSION TITLE
I just can't get tilemaker to work...

### DISCUSSION THREAD

**Discussion by martinsnajdr** [systemed/tilemaker#528] [Q&A]
Hello,

probably missing something simple but...

At first I have downloaded the macOS binary (2.4.0), cd to build folder, `./tilemaker --help` = getting permissions denied error.

Then I have realized the binary is labeled as "10.15", I'm using the latest version of macOS. So I have tried to build it myself. Downloaded source code, installed all the required packages using brew, "make" = getting a lot of errors like `/usr/local/include/absl/container/internal/compressed_tuple.h:224:46: error: no member named 'in_place' in namespace 'absl'; did you mean 'isspace'?` 

So I have used docker, but after running the command (with my parameters) `docker run -v /Users/Local/Downloads/:/srv -i -t --rm tilemaker /srv/germany-latest.osm.pbf --output=/srv/germany.mbtiles` it does something, it reads the .osm.pbf file, outputs something like `Block xxx/12030 ways 0 relations 0`, but it never outputs/saves the mbtiles file anywhere...

Thank you for any advice.

**Comment by systemed** [comment]
For a build from source on MacOS at the moment you'll need to use cmake - unfortunately Homebrew/Google have broken the protobuf dependency. See the instructions for "Using cmake" in INSTALL.md.

**Reply by martinsnajdr** [reply]
Thank you for a quick answer, but unfortunately I'm getting the same errors (`cmake ..` seems to work), but the `make` command throws same set of errors...

I don't mind using docker or prebuild binaries, but those are not working for me too...
